hash.ts

```diff
@@ -10,6 +10,54 @@ export type Description = Record<string, string | string[]> & {
   eNS?: Record<string, Record<string, string>>;
 };
 
+function findAccessPointReferences(e: Element): Element[] {
+  let serverAt = e.querySelector(':scope>ServerAt, :scope>Server');
+  let apName = serverAt?.getAttribute('apName');
+  while (serverAt && serverAt.tagName !== 'Server') {
+    serverAt =
+      serverAt
+        .closest('IED')
+        ?.querySelector(
+          `:scope>AccessPoint[name="${apName}"]>Server, :scope>AccessPoint[name="${apName}"]>ServerAt`,
+        ) ?? null;
```

Comment:
There is an issue with selectors like this not respecting filter criteria.

Reply:
Done.